README.md

```diff
@@ -15,3 +15,29 @@ Witzige, originelle Inhalte, die nicht weitläufig verbreitet sind.
 ## Mitmachen
 
 [Gabeln](https://github.com/Lodifice/praedikat-wertvoll/fork) und [Zugbehren erstellen](https://github.com/lodifice/praedikat/compare).
+
+### Weiterentwicklung der visuellen Aufbereitung
+
+Diese visuelle Aufbereitung für das WWW wird mithilfe des kleinen Programms `jekyll` erstellt.
+Um die Schablonen weiter zu bringen, ist eine lokale Installation des Programms `jekyll` inklusive der Abhängigkeiten notwendig.
+
+#### Docker
+
+Um sich nicht eine komplette Umgebung mit Rubin, Edelsteinen und Bündel aufsetzen zu müssen, kann man sich Hilfe bei `Docker` holen.
+Das Abbild `jekyll/jekyll:VERSIONSNUMMER` funktioniert wunderbar, um eine lokal begrenzte Umgebung zu besitzen.
+Für Github Pages Umgebungen ist `jekyll` in der Version 3.9 gesetzt, weswegen man `jekyll/jekyll:3` benutzen sollte.
```

Comment:
Bitte Eigennamen nicht ins Englische Übersetzen und Deppenleerzeichen vermeiden! Vorschlag: Deppendrehkreuzseitenumgebungen.

Reply:
You are sending conflicting signals here.
Either I should not translate or I should.

Reply:
Oh yes, there it is! I finally retained sanity and was able to comprehend the ingeniousness of my statement! 

There is no contradiction. I asked you to not translate proper names like Deppendrehkreuz into English.

Reply:
There you go buddy.
c2f67b431e3feea8908379a2e7a3d78719affa37
This one is on you, answering to a resolved thread.